frontend/site.py

```diff
@@ -26,5 +26,5 @@
     draw_evaluation()
 elif selection == 'Model selection':
     draw_model_selection(selected_models)
-elif selection == "Data corpuses":
-    draw_data_corpuses()
+elif selection == "Data corpora":
```

Comment:
To jest fajne, tylko zrobiłeś bug'a, bo w `sidebar.py` nie zmieniłeś :)
Siedziałem 15 minut by zrozumieć czemu mi się widok z korpusami nie wyświetla, poprawiłem już to ;)

Reply:
Sooorki :D Nie działało mi wyszukiwanie w plikach, więc ctr-f robiłem i zmieniałem wszędzie gdzie było 'corpuses'